File: week 7/recipes callback/recipes/Recipe.swift
//
//  Recipe.swift
//  recipes
//
//  Created by Aileen Pierce
//  Copyright Aileen Pierce. All rights reserved.
//

import Foundation
import Firebase

struct Recipe {
    var id : String
    var name: String
    var url: String

    init(id: String, name: String, url: String){
        self.id = id
        self.name = name
        self.url = url
    }
}

File: week 15 Firebase/Recipes/app/src/main/java/com/example/aileen/recipes/Recipe.java
package com.example.aileen.recipes;

/**
 * Created by aileen
 */

public class Recipe {
    private String id;
    private String name;
    private String url;

    public Recipe(){
        // Default constructor required for calls to DataSnapshot.getValue(Recipe.class)
    }

    public Recipe(String newid, String newName, String newURL){
        id = newid;
        name = newName;
        url = newURL;
    }

    public String getId() {
        return id;
    }

    public String getName(){
        return name;
    }

    public String geturl(){
        return url;
    }
}
